source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -90,26 +90,17 @@ def move(self, unit, direction, endPoint=None):
 
 	def _move(self, unit, direction, endPoint=None):
 		"Perform a move without respect to bounding."
-		return super(consoleUIATextInfo, self).move(unit, direction, endPoint)
+		return super(TextInfo, self).move(unit, direction, endPoint)
 
 	def __ne__(self, other):
 		"""Support more accurate caret move detection."""
 		return not self == other
 
-	def _get_text(self):
-		# #10036: return a space if the text range is empty.
-		# Consoles don't actually store spaces, the character is merely left blank.
-		res = super(consoleUIATextInfo, self)._get_text()
-		if not res:
-			return ' '
-		else:
-			return res
-
 
-class consoleUIATextInfoPre21H1(consoleUIATextInfo):
-	"""Fixes expand/collapse on end inclusive UIA text ranges, uses rangeFromPoint
-	instead of broken GetVisibleRanges for bounding, and implements word
-	movement support."""
+class TextInfoWorkaroundEndInclusive(TextInfo):
```

Comment:
The problem with using `TextInfo` as a name is that code like this becomes quite ambiguous for a reader. I think it is unnecessarily ambiguous. Consistency is worthwhile if the convention scales, I'm not sure this convention really makes anything better, other than allowing the developer a way out of coming up with a descriptive name.

Code is read many more times than it is written. Better to optimize for the reader IMO.

Reply:
When I say ambiguous, I mean that you have to be aware that there is a class within this module called TextInfo, and check that the import doesn't do `from textInfos import TextInfo`.

Reply:
OK, the name is now `ConsoleUIATextInfo` (with capitalization consistent with the rest of NVDA).

In the long term (i.e. once microsoft/terminal#6986 has been resolved), I hope to transition to just using the standard `UIATextInfo` implementation for consoles where `isImprovedTextRangeAvailable`... but that's in a follow-up PR.